### PR TITLE
added last checked timestamp

### LINKED ISSUE #44: New feature request: ability to display timestamp when insights data was 'last checked'
It is needed to display the 'last checked' field for given cluster on UI, so we need to be able to store timestamp in a storage.

### PR DESCRIPTION
**Pull request by Sergey1011010**
# Description

added last checked timestamp

Fixes #44 

## Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)
- Refactor (refactoring code, removing useless files)

## Testing steps

`make test` & `make integration_tests`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -29,7 +29,7 @@ cyclo: ## Run gocyclo
 style: fmt vet lint cyclo ## Run all the formatting related commands (fmt, vet, lint, cyclo)
 
 run: clean build ## Build the project and executes the binary
-	./insights-operator-aggregator 
+	./insights-results-aggregator
```
Comment (tisnik) [inline Makefile]: good catch

File: storage/storage_test.go [hunk 2]
```diff
@@ -57,7 +58,7 @@ func checkReportForCluster(t *testing.T, storage storage.Storage, orgID types.Or
 }
 
 func writeReportForCluster(t *testing.T, storage storage.Storage, orgID types.OrgID, clusterName types.ClusterName, report types.ClusterReport) {
-	err := storage.WriteReportForCluster(orgID, clusterName, report)
+	err := storage.WriteReportForCluster(orgID, clusterName, report, time.Now())
```
Comment (tisnik) [inline storage/storage_test.go]: not sure whether you want to `time.Now().UTC()`

**Comment by tisnik** [comment]
@Sergey1011010 prob. it might be worth to update scripts in `local_storage`

File: storage/storage_test.go (hunk 2 above)
Comment (Sergey1011010) [reply storage/storage_test.go]: looks like it won't matter because we will have timestamp in db, but I'm not sure too

Comment (tisnik) [reply storage/storage_test.go]: I see, TY

**Pull request merged**